cmake/Platform/Windows/Packaging_windows.cmake

```diff
@@ -17,6 +17,10 @@ else()
     return()
 endif()
 
+# IMPORTANT: CPACK_WIX_ROOT is a built-in variable that is required to propagate the path supplied
+# via command line down to the cpack internals
+set(CPACK_WIX_ROOT ${LY_INSTALLER_WIX_ROOT})
```

Comment:
ah, sorry about that, that was me, CPACK_WIX_ROOT was being declared as a cache variable so it was showing in the cmake-gui but not categorized properly.
this set will prevent it from being in the cmake-gui, thanks